packages/tools/devtools/devtools-view/jest.setup.cjs

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+HTMLCanvasElement.prototype.getContext = jest.fn();
```

Comment:
For my education, what does this do?

Reply:
I was running into some canvas errors when I was first testing and saw that it needed to be mocked like this for the tests to work. I might have fixed that later on by cleaning and rebuilding the code. Tests pass even after removing this code locally, so I'll leave it out